src/emulator/functionsEmulator.ts

```diff
@@ -584,7 +583,7 @@ export class FunctionsEmulator implements EmulatorInstance {
       } else {
         this.logger.log(
           "WARN",
-          `Trigger trigger "${definition.name}" has has neither "httpsTrigger" or "eventTrigger" member`
+          `Trigger "${definition.name}" has neither "httpsTrigger" or "eventTrigger" member`
```

Comment:
Good point - updated.

Note that we have some weird dance in parsing the trigger in the emluator that converts scheduled triggers into event triggers. This isn't great, and I want us to eventually move to a point where Functions Emulator speaks Endpoint instead of this "EmulatedTrigger" variant, but I'm pushing that out till another time.